src/components/Menu/CreateParty.vue

```diff
@@ -34,22 +34,22 @@ export default {
   },
   methods: {
     ...mapActions(['setParty']),
+    async createParty() {
+      const URL_PLAYER = `${this.$env.VUE_APP_API_BASE_URL}/party/${this.partyId}/add-player`
+      const URL_PARTY = `${this.$env.VUE_APP_API_BASE_URL}/party/${this.partyId}/details`
+      try {
+        await this.$http.post(URL_PLAYER, {userId: this.user._id})
+
+        const party = await this.$http.get(URL_PARTY)
+        this.setParty(party.data)
+        await this.$router.push({name: 'Party', params: {partyId: this.partyId}})
```

Comment:
The expected behavior was just to put this part at the click of the button but the rest was well in the mounted.